flutter/lib/sentry_flutter.dart

```diff
@@ -1,5 +1,7 @@
-// ignore: invalid_export_of_internal_element
 /// A Flutter client for Sentry.io crash reporting.
+library sentry_flutter;
+
+// ignore: invalid_export_of_internal_element
```

Comment:
@denrase I was making linting changes so the workflow doesn't fail. Do you know if adding `library sentry_flutter;` has any effect / is safe?

Reply:
https://dart.dev/tools/linter-rules/dangling_library_doc_comments